### DISCUSSION TITLE
Installing onto a Truenas installed PMS?

### DISCUSSION THREAD

**Discussion by Gimpymoo** [Taxel/PlexTraktSync#1937] [Support] [answered]
Apologies if not the correct place to ask such a question but is it possible to install this into a Truenas installed PMS?

Many thanks.

**Comment by glensc** [comment, marked as answer]
What is Truenas? what is PMS?

Generic instructions are either install via docker image or install Python and use pipx.

**Comment by glensc** [comment]
user disappeared. closing
